### PR TITLE
[PLATFORM-189] Show relative canvas updated time.

### PR DESCRIPTION
**Pull request by timoxley**
Shows relative "updated" time in bottom right corner of editor. Updates every ~10s.

It'd be possible to do this relative time setup with `moment` but that's a massive dependency and we should be phasing it out in favour of `date-fns` or `luxon`.

### CODE REVIEW

File: app/src/editor/components/Status.pcss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.status {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  color: #CDCDCD;
+  font-size: 12px;
+  text-align: right;
+  letter-spacing: 0;
+  z-index: 12;
+  margin: 13px 40px;
```
[the remark this answers is not in the document]
Comment (timoxley) [reply app/src/editor/components/Status.pcss]: even better: `user-select: none` 👍  good idea

File: app/src/editor/components/Status.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+
+import styles from './Status.pcss'
+
+function format(diff, divisor, unit, prev) {
```
[the remark this answers is not in the document]
Comment (timoxley) [reply app/src/editor/components/Status.jsx]: Agreed that would be a good place for it though it's probably premature to make something shared before it's actually shared.

[the remark this answers is not in the document]
Comment (timoxley) [reply app/src/editor/components/Status.jsx]: I guess the downside of that approach is it requires you to hunt in multiple locations for reusable logic/components, but the upside is shared won't be full of unshared code.

[the remark this answers is not in the document]
Comment (juhah) [reply app/src/editor/components/Status.jsx]: Yep, perhaps better to move it if / when there is a need for this.

File: app/src/editor/components/Status.pcss (hunk 1 above)
[the remark this answers is not in the document]
Comment (timoxley) [reply app/src/editor/components/Status.pcss]: Fixed in d2ba4c4

**Pull request merged**